src/components/DocumentationLayout.vue

```diff
@@ -150,6 +151,7 @@ export default {
       sidenavHiddenOnLarge: storage.get(NAVIGATOR_HIDDEN_ON_LARGE_KEY, false),
       showQuickNavigationModal: false,
       BreakpointName,
+      state: IndexStore.state,
```

Comment:
Minor thing, but I might suggest naming this `indexState` or something since `state` is kind of generic on its own—especially since all the fields within `data` are technically state that may change over time.